### ISSUE TITLE
Bug with screen >= 4.5.0 on Debian 9

### ISSUE THREAD

**Issue by soyelraul** [crazy-max/csgo-server-launcher#38]
hey i updated to debian 9 and now this script is not working correctly, when i try to start a server, it throws this and the server don't start:

```
#####@####_#######:~$ /etc/init.d/csgo-server-launcher start
Starting csgo_awp...
Create runscript file '/home/##########/steamcmd/runscript_csgo_awp' for autoupdate...
Start command : -nobreakpad -game csgo -console -usercon -secure -autoupdate -steam_dir /home/##########/steamcmd -steamcmd_script /home/##########/steamcmd/runscript_csgo_awp -maxplayers_override 33 -tickrate 128 +hostport 27015 +ip **.**.**.* +net_public_adr **.**.**.* -nohltv -nobots +sv_pure 0 +game_type 0 +game_mode 0 +map awp_lego_2017_1  +sv_setsteamaccount **************************
Use: screen [-opts] [cmd [args]]
 or: screen -r [host.tty]

Options:
-4            Resolve hostnames only to IPv4 addresses.
-6            Resolve hostnames only to IPv6 addresses.
-a            Force all capabilities into each window's termcap.
-A -[r|R]     Adapt all windows to the new display width & height.
-c file       Read configuration file instead of '.screenrc'.
-d (-r)       Detach the elsewhere running screen (and reattach here).
-dmS name     Start as daemon: Screen session in detached mode.
-D (-r)       Detach and logout remote (and reattach here).
-D -RR        Do whatever is needed to get a screen session.
-e xy         Change command characters.
-f            Flow control on, -fn = off, -fa = auto.
-h lines      Set the size of the scrollback history buffer.
-i            Interrupt output sooner when flow control is on.
-l            Login mode on (update /var/run/utmp), -ln = off.
-ls [match]   or
-list         Do nothing, just list our SockDir [on possible matches].
-L [file]     Turn on output logging. Set logfile name.
-m            ignore $STY variable, do create a new screen session.
-O            Choose optimal output rather than exact vt100 emulation.
-p window     Preselect the named window if it exists.
-q            Quiet startup. Exits with non-zero return code if unsuccessful.
-Q            Commands will send the response to the stdout of the querying process.
-r [session]  Reattach to a detached screen process.
-R            Reattach if possible, otherwise start a new session.
-s shell      Shell to execute rather than $SHELL.
-S sockname   Name this session <pid>.sockname instead of <pid>.<tty>.<host>.
-t title      Set title. (window's name).
-T term       Use term as $TERM for windows, rather than "screen".
-U            Tell screen to use UTF-8 encoding.
-v            Print "Screen version 4.05.00 (GNU) 10-Dec-16".
-wipe [match] Do nothing, just clean up SockDir [on possible matches].
-x            Attach to a not detached screen. (Multi display mode).
-X            Execute <cmd> as a screen command in the specified session.

Error: Unknown option -nobreakpad
csgo_awp started successfully
```

**Comment by crazy-max** [comment]
Hi @soyelraul,

I will investigate on this issue, i keep you in touch.

**Comment by crazy-max** [comment]
On Debian 9, screen has been updated to 4.05.00 (10-Dec-16) and changes have been made with the `-L` flag. Will be solved asap.

**Comment by crazy-max** [comment]
Solved on 1.12.3 release
